packages/chopsticks/src/context.ts

```diff
@@ -19,6 +20,19 @@ export const setupContext = async (argv: Config, overrideParent = false) => {
     maxMemoryBlockCount: argv['max-memory-block-count'],
   })
 
+  // load block from db
+  if (chain.db) {
+    if (argv.resume) {
+      const blockData = await chain.db.getRepository(BlockEntity).findOne({ where: {}, order: { number: 'desc' } })
+      if (blockData) {
+        const block = await chain.loadBlockFromDB(blockData?.number)
+        block && (await chain.setHead(block))
+      }
+    } else {
+      await chain.db.getRepository(BlockEntity).clear()
```

Comment:
why clear?

Reply:
to give them a fresh start? and also free some space

Reply:
it is always bad to delete thing without user explicitly ask for it. we should have a purge command for that